modules/tinymce/src/themes/silver/main/ts/alien/NotificationManagerImpl.ts

```diff
@@ -20,40 +20,31 @@ interface Extras {
   readonly backstage: UiFactoryBackstage;
 }
 
-type Location = 'tc-tc' | 'bc-bc' | 'bc-tc' | 'tc-bc';
-
 export default (editor: Editor, extras: Extras, uiMothership: Gui.GuiSystem): NotificationManagerImpl => {
   const sharedBackstage = extras.backstage.shared;
 
-  const getLayoutDirection = (rel: Location) => {
-    switch (rel) {
-      case 'bc-bc':
-        return LayoutInset.south;
-      case 'tc-tc':
-        return LayoutInset.north;
-      case 'tc-bc':
-        return Layout.north;
-      case 'bc-tc':
-      default:
-        return Layout.south;
-    }
-  };
-
-  const reposition = (notifications: NotificationApi[]) => {
-    if (notifications.length > 0) {
-      Arr.each(notifications, (notification, index) => {
-        if (index === 0) {
-          notification.moveRel(null, 'banner');
-        } else {
-          notification.moveRel(notifications[index - 1].getEl(), 'bc-tc');
-        }
-      });
-    }
+  const getBounds = () => {
+    /* Attempt to ensure that the notifications render below the top of the header and between
+     * whichever is the larger between the bottom of the content area and the bottom of the viewport
+     *
+     * Note: This isn't perfect, but without being able to use docking and associate the notifications
+     * together due to the `moveTo` and `moveRel` APIs then we're a bit stuck and a proper solution
+     * will have to be done in TinyMCE 6.
```

Comment:
6 is still correct; this breaking change opens up the option to do it. We ran out of time. I'll update the wording to make this clearer.

Reply:
Right maybe just drop the version number. Since to me having a reference to a major like that hints it will be a breaking change. Like a to do we can now use the docking behavior since moveRel and move to was removed.

Reply:
I just changed it, is that better?